### ISSUE TITLE
`eval()` in class constructor / method which uses `super` causes illegal output

### ISSUE THREAD

**Issue by overlookmotel** [overlookmotel/livepack#234]
Input:

```js
class X {}
export default class extends X {
  constructor() {
    super();
    eval('0');
  }
};
```

Output:

```js
const ObjectSetPrototypeOf = Object.setPrototypeOf,
  X = class X {},
  index = (
    (X, super) => super = ( 0, class {
      constructor() {
        eval('0');
        return Reflect.construct(Object.getPrototypeOf(super), [], super);
      }
    } )
  )(X);
ObjectSetPrototypeOf(index, X);
ObjectSetPrototypeOf(index.prototype, X.prototype);
export default index;
```

The output contains an illegal var called `super`. This var name is used internally in Livepack to denote the "home" class of a method. `super` is an illegal var name, so it always gets changed later. But in this case `eval()` has resulted in var names being frozen. An exception should be made for `super`.

Same problem for `eval()` in class methods which use `super.method()`. Also same problem for Object methods:

```js
export default {
  toString() {
    eval('0');
    return super.toString();
  }
};
```

**Comment by overlookmotel** [comment]
If there's a statement after `super()`, also get a var illegally named `this`, for same reasons as above:

Input:

```js
class X {}
export default (0, class extends X {
  constructor() {
    super();
    eval('0');
  }
});
```

Output:

```js
const ObjectSetPrototypeOf = Object.setPrototypeOf,
  X = class X {},
  index = (
    (X, super) => super = ( 0, class {
      constructor(module, exports) {
        const this = Reflect.construct(Object.getPrototypeOf(super), [], super);
        eval('0');
        return this;
      }
    } )
  )(X);
ObjectSetPrototypeOf(index, X);
ObjectSetPrototypeOf(index.prototype, X.prototype);
export default index;
```

**Comment by overlookmotel** [comment]
Best solution to all of this is not to transpile `super` (#48).

Removing transpiled `super` from class constructors (https://github.com/overlookmotel/livepack/issues/48#issuecomment-887731490) would at least prevent the problem with `this` in comment above. Problem with var called `super` would remain for class/object methods.

**Comment by overlookmotel** [comment]
Fixed in 2ec0f33.
